docs/mission.md

```diff
@@ -0,0 +1,47 @@
+
+---
+
+## Flax: The missing parts for neural networks with JAX
+
+JAX is an excellent basis for current research in many fields in numerical computing--particularly with neural networks. To directly serve various classes of researchers who use neural networks with JAX, it is necessary to have an ecosystem of libraries, carefully designed to build around JAX's philosophy.
+
+We, on the Flax team, aim to ensure the JAX ecosystem has the necessary tools and guidance to facilitate and maximize this new space of research directions in the form of libraries, integrated examples for both educational purposes, as well as ready-to-fork reference implementations, guides, support channels, and best practices.
+
+Flax aims to support researchers' needs, while exposing the full power of JAX, rather than encapsulating it away. We think that this is the best path towards ground-breaking research that builds on JAX's unique capabilities. Some recent examples of innovative research projects on top of Flax include [NetKet](https://www.netket.org/) for many-body quantum systems, [T5X](https://github.com/google-research/t5x) for very large scale language model and [Brax](https://github.com/google/brax) for differentiable on-device rigid body simulations. 
+
+It is challenging to design a library that is ergonomic for researchers, while building on a purely functional library like JAX, which may be confusing to some. But we see this challenge as **an opportunity**. How do you design a smooth user experience where the complexity is progressively disclosed? How do you help users avoid "silent footguns", such as accidentally mutating batch norm stats during prediction? How do you make sure errors guide users to the right solution, even when there may be multiple correct ones? How do you design examples and tutorials that are simple enough to be educational, while being good enough to act as forkable starter codebases? These are the types of questions that inform our design process.
+
+As members of the Flax team, we spend a majority of our time on **community engagement**. To make sure what we build is delightful for our current and future users, we hold design discussions in the open. This helps the community share ideas and engage with Flax core developers directly, while we learn from each others' complementary experiences and integrate the feedback. We [actively engage](https://github.com/google/flax/issues/208) with users via GitHub issues, discussions and pull requests. We share [design notes](https://flax.readthedocs.io/en/latest/design_notes/linen_design_principles.html), [philosophy](https://flax.readthedocs.io/en/latest/philosophy.html), and [recommended best practices](https://flax.readthedocs.io/en/latest/howtos.html) with the community. We improve our APIs based on feedback, while supporting our users through any transition—take a look at how we did it for our recent ["Linen" rewrite](https://github.com/google/flax/tree/main/flax/linen), as an example.
```

Comment:
I propose we change the link on "actively engage" to https://github.com/google/flax/pull/1011, as a more recent example.